### ISSUE TITLE
Server plugin manager

### ISSUE THREAD

**Issue by taylortom** [adaptlearning/adapt_authoring#1213]
One for the (probably fairly distant) future, we could do with management functionality in the UI for server plugins (things like authentication types, storage types, and other custom plugins).

**Comment by brian-learningpool** [comment]
I agree.  Much of the `config.json` could move into the database and be rendered in the UI.
